dotcom-rendering/src/web/lib/decideFormat.ts

```diff
@@ -2,8 +2,10 @@ import { decideDesign } from './decideDesign';
 import { decideDisplay } from './decideDisplay';
 import { decideTheme } from './decideTheme';
 
-export const decideFormat = (format: Partial<CAPIFormat>): ArticleFormat => ({
-	display: decideDisplay(format),
-	theme: decideTheme(format),
-	design: decideDesign(format),
+export const decideFormat = (
+	format: CAPIFormat | undefined,
+): ArticleFormat => ({
+	display: decideDisplay(format?.display),
+	theme: decideTheme(format?.theme),
+	design: decideDesign(format?.design, format?.display),
```

Comment:
🤔 why are we accepting an undefined `format` object? This feels... wrong? I guess the reason is the snap might not have `format` defined but that feels like a problem of the snap, rather than something that `decideFormat` should have to deal with.

Ideally we would fix the snap to be more consistent with predictable structured data but in the event this isn't possible I think we should default it to a standard format property and then pass *that* into our decide functions, keeping these props as required - because they really should be required.